velox/exec/Unnest.cpp

```diff
@@ -111,56 +112,111 @@ RowVectorPtr Unnest::getOutput() {
   }
 
   const auto size = input_->size();
-  const auto maxOutputSize = outputBatchRows();
 
   // Limit the number of input rows to keep output batch size within
-  // 'maxOutputSize' if possible. Process each input row fully. Do not break
-  // single row's output into multiple batches.
-  vector_size_t numInput = 0;
+  // 'maxOutputSize' if possible. Not process each input row fully when a
```

Comment:
Is it the case that only the first and last rows might not be fully processed? Perhaps clarify that here.